src/python/pose_format/bin/directory.py

```diff
@@ -79,6 +111,21 @@ def get_corresponding_pose_path(video_path: Path, keep_video_suffixes: bool = Fa
     return video_path.with_suffix(".pose")
 
 
+def process_video(vid_path: Path, keep_video_suffixes: bool, pose_format: str, additional_config: dict) -> bool:
+    print(f'Estimating {vid_path} on CPU {psutil.Process().cpu_num()}')
```

Comment:
We should probably switch to logging at some point rather than print statements, but apparently combining that with multiprocessing [can be tricky.](https://stackoverflow.com/questions/641420/how-should-i-log-while-using-multiprocessing-in-python)

https://docs.python.org/3/howto/logging-cookbook.html#logging-to-a-single-file-from-multiple-processes seems easiest.